ingesters/s3Ingester/config.go

```diff
@@ -95,6 +97,7 @@ func GetConfig(path, overlayPath string) (*cfgType, error) {
 		IngestConfig:         cr.Global.IngestConfig,
 		Attach:               cr.Attach,
 		State_Store_Location: cr.Global.State_Store_Location,
+		Worker_Pool_Size:     cr.Global.Worker_Pool_Size,
```

Comment:
init cr.Global.Worker_Pool_Size to one prior to reading the file into the cr structure.